app/components/vistk-scatterplot.js

```diff
@@ -88,10 +86,26 @@ export default Ember.Component.extend({
           type: 'line_horizontal',
           filter: function(d, i) {
            return typeof eci !== 'undefined' && i === 0;
-         },
+          },
           offset_y: function(d, i, vars) {
             return -(vars.y_scale[0]['func'](d[vars.var_y]) - vars.y_scale[0]['func'](eci));
           }
+        }, {
+          type: 'text',
+          filter: function(d, i) {
+           return typeof eci !== 'undefined' && i === 0;
+          },
+          text: function(d, i) {
+            var label = lang === 'en_EN' ? 'Average complexity': 'Complejidad media';
+            return label + ': ' + format(eci);
+          },
+          text_anchor: 'end',
+          offset_y: function(d, i, vars) {
+            return -(vars.y_scale[0]['func'](d[vars.var_y]) - vars.y_scale[0]['func'](eci)) - 10;
```

Comment:
Oh I see, so the `label` is formatted.  But the offset isn't, should the `eci` passed here be the same as the input for the label?